### ISSUE TITLE
Add global menu icon.

### ISSUE THREAD

**Issue by CavaleriOmar** [zzag/material-decoration#2]
Is it possible add an icon for the global menù on the left of the titlebar?

**Comment by zzag** [comment]
Yes, it's possible. I just didn't do it. Only three button types are implemented - minimize, maximize, and close. Buttons like shade, keep above and so on are not implemented.

**Comment by zzag** [comment]
Please notice that this decoration theme is not meant to be used by ordinary people. That's why I put "(please don't use it)".
